components/dash-core-components/src/components/Loading.react.js

```diff
@@ -148,6 +149,7 @@ const Loading = ({
                 {showSpinner &&
                     (custom_spinner || (
                         <Spinner
+                            id={id}
```

Comment:
The id should be on the outer div,  the spinners https://github.com/plotly/dash/blob/4692acf61f3e282950a8d1014b152176addb2aec/components/dash-core-components/src/fragments/Loading/spinners/ don't have the id prop so it wouldn't render on the DOM.

Reply:
Ah I see, good catch 😄 

Changed in 9dcf243